tests/test_status.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2018 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+import io
+import os
+import sys
+
+import generate_status_reference_data as gen
+import signac
+from test_project import redirect_stderr, redirect_stdout
+
+
+def test_print_status():
+
+    # Force asserts to show the full file when failures occur.
+    # Useful to debug errors that arise.
+
```

Comment:
I don't know what this means. Let's remove it unless you know. Also make this change to the template tests if relevant.
```suggestion
```